Source/UnitTests/Core/DSP/DSPTestText.cpp

```diff
@@ -188,223 +194,223 @@ MEM_LO:			equ		0x0f7F
 
 ;	call dump_memory
 ;	call	send_back
-	
-	
+
+
 ; 0x041e
-;	
+;
 
-    cw 0x00de 
-    cw 0x03f1 
+    cw 0x00de
+    cw 0x03f1
     call	send_back
-    
-    cw 0x0200 
-    cw 0x0a60 
+
+    cw 0x0200
+    cw 0x0a60
     call	send_back
-    
-    cw 0x1c7e 
+
+    cw 0x1c7e
     call	send_back
-    
-    cw 0x8100 
+
+    cw 0x8100
     call	send_back
-    
-    cw 0x8900 
+
+    cw 0x8900
     call	send_back
-    
-    cw 0x009f    
-    cw 0x00a0 
+
+    cw 0x009f
+    cw 0x00a0
     call	send_back
-    
-    cw 0x00de 
-    cw 0x03f1 
+
+    cw 0x00de
+    cw 0x03f1
     call	send_back
-    
-    cw 0x5d00 
+
+    cw 0x5d00
     call	send_back
-    
-    cw 0x0e50 
+
+    cw 0x0e50
     call	send_back
-    
-    cw 0x0750 
+
+    cw 0x0750
     call	send_back
-    
-    cw 0x0270 
+
+    cw 0x0270
     call	send_back
-    
-    cw 0x5d00 
+
+    cw 0x5d00
     call	send_back
-    
-    cw 0x00da    
-    cw 0x03f2 
+
+    cw 0x00da
+    cw 0x03f2
     call	send_back
-        
-    cw 0x8600 
+
+    cw 0x8600
     call	send_back
-     
-    JNS g_0c4d
-;		cw 0x0290 
-;   cw 0x0c4d 
+
+    JNZ g_0c4d
```

Comment:
Ah okay. I changed it because `JNS` isn't accepted by Dolphin, figured it was a typo.